src/Components/ActivationKey/__tests__/ActivationKey.test.js

```diff
@@ -108,11 +108,9 @@ describe('ActivationKey', () => {
   });
 
   it('renders correctly', async () => {
-    render(<PageContainer />);
+    const { container } = render(<PageContainer />);
     await waitFor(() => expect(useUser).toHaveBeenCalledTimes(1));
-    expect(screen.getByText('System Purpose')).toBeInTheDocument();
-    expect(screen.getByText('Workload')).toBeInTheDocument();
-    expect(screen.getByText('Additional repositories')).toBeInTheDocument();
+    expect(container).toMatchSnapshot();
   });
 
```

Comment:
We shouldn't have to switch this test to a snapshot test - it makes me nervous to see a test change when functionality shouldn't. That makes me think the functionality _is_ changing.